SUPPORTED_LANGUAGES.md

```diff
@@ -126,6 +126,7 @@ The table below shows the full list of languages (and corresponding classes/alia
 | Lasso                   | lasso, ls, lassoscript |         |
 | Less                    | less                   |         |
 | LDIF                    | ldif                   |         |
+| Liquid                  | liquid                 | [highlightjs-liquid](https://github.com/leanprover-community/highlightjs-liquid) |
```

Comment:
Reason this wouldn't link to the new repo here?

Reply:
ugh I thought I swapped the link, that was a placeholder.